src/components/OrderDataTable.js

```diff
@@ -16,8 +16,12 @@ class OrderDataTable extends React.Component {
                   id: "Pick Up Date",
                   Header: "Pick Up Date",
                   accessor: row => `${row.PickUpDate}`,
+                  /*Arrow function that returns true if Date values are
```

Comment:
again, this is the 3rd iteration that you haven't changed these to proper format, see trevor's comments.